### PR TITLE
added severity and labels

### PR DESCRIPTION
**Pull request by cr-alan**
#### What does this PR do?
This adds severity to the Title of the jira
Also adds the capability to configure labels for jiras created with this tool.

#### Where should the reviewer start?
Lines were added/changed in SecurityAlertIssue.php
Readme was updated.

#### Any background context you want to provide?
Severity is a critical component for prioritization.  This deserves a place in the title.  
Labels would be a great add-on.  

#### Is this related to processing of personal data?
No

### CODE REVIEW

**Comment by carlossless** [comment]
Would love to see this introduced! :)

File: README.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ It has some required and some optional settings, which are passed to the action
 - `JIRA_PROJECT`: The project key for the Jira project where issues should be created, eg `TEST` or `ABC`. (**REQUIRED**)
 - `JIRA_ISSUE_TYPE`: Type of issue to create, e.g. `Security`. Defaults to `Bug`. (*Optional*)
 - `JIRA_WATCHERS`: Jira users to add as watchers to tickets. Separate multiple watchers with comma (no spaces).
+- `JIRA_LABELS`: Jira labels to add to tickets. Separate multiple labels with comma (no spaces).
```
Comment (achton) [inline README.md]:
```suggestion
- `JIRA_ISSUE_LABELS`: Jira labels to add to tickets. Separate multiple labels with comma (no spaces).
```

**Comment by cr-alan** [comment]
Curious to see if this will get merged in?

File: src/SecurityAlertIssue.php [hunk 2]
```diff
@@ -87,8 +88,14 @@ public function __construct(array $data)
...
         $this->setKeyLabel($this->uniqueId());
-        $this->setTitle("{$this->package} ({$safeVersion})");
+        $this->setTitle("{$this->package} ({$safeVersion}) - {$this->severity}");
         $this->setBody($body);
+
+        $labels = \getenv('JIRA_ISSUE_LABELS');
+        foreach (\explode(',', $labels) as $label){
+            $this->setKeyLabel($label);
+        }
+
```
Comment (xendk) [inline src/SecurityAlertIssue.php]:
```suggestion

        foreach (\explode(',', $labels) as $label) {
            $this->setKeyLabel($label);
        }
```

File: src/SecurityAlertIssue.php [hunk 3]
```diff
@@ -87,8 +88,14 @@ public function __construct(array $data)
 
         $this->setKeyLabel($githubRepo);
         $this->setKeyLabel($this->uniqueId());
-        $this->setTitle("{$this->package} ({$safeVersion})");
+        $this->setTitle("{$this->package} ({$safeVersion}) - {$this->severity}");
         $this->setBody($body);
+
+        $labels = \getenv('JIRA_ISSUE_LABELS');
+        foreach (\explode(',', $labels) as $label){
```
Comment (xendk) [inline src/SecurityAlertIssue.php]: `getenv()` might return `false` if the variable doesn't exist, so you should wrap the foreach in a `if ($labels) {`.

File: src/SecurityAlertIssue.php [hunk 4]
```diff
@@ -47,6 +47,7 @@ public function __construct(array $data)
         $this->vulnerableVersionRange = $data['securityVulnerability']['vulnerableVersionRange'];
         $this->manifestPath = \pathinfo($data['vulnerableManifestPath'], \PATHINFO_DIRNAME);
         $this->id = $data['securityVulnerability']['advisory']['ghsaId'];
+        $this->severity = $data['securityVulnerability']['severity'];
```
Comment (xendk) [inline src/SecurityAlertIssue.php:55]: Please define the property in the section above.

**Review by xendk — CHANGES_REQUESTED** [review]
Basically looks good. I've let phpcs and phpstan run, and there's a few style issues to fix. I've summarized them in comments.

**Comment by drchiodo** [comment]
up

**Comment by xendk** [comment]
OK, one PHPCS notice left. I'll merge and fix it afterwards.

**Pull request merged**